web/src/app/selection/MaterialSelect.tsx

```diff
@@ -142,11 +145,12 @@ export default function MaterialSelect(
       data-cy='material-select'
       data-cy-ready={!isLoading}
       classes={customCSS}
-      {...multi}
+      multiple={multiple}
       value={value}
       inputValue={inputValue}
       disableClearable={required}
       disabled={disabled}
+      filterSelectedOptions
```

Comment:
yeah, this was intentional because the only time the selected option would show in the dropdown as well is if you were to pick one of the first five options without ever searching anything. so i opted for the consistency between multiple/single selects since it helped get around the bugs a bit easier

Reply:
What bugs are we talking about? With single selection it feels off with things like this:
![image](https://user-images.githubusercontent.com/595010/165130000-4a5f0c62-19dc-4261-9af0-c42f9202b50b.png)

It also means if you explicitly search for a user, they will be omitted if they happen to be selected:

![image](https://user-images.githubusercontent.com/595010/165130391-af868457-a920-43fd-a0b6-3637cc40e3bf.png)

It's not an issue for multi because the chip displays, but in single select mode there is no indicator

Reply:
gotcha, the issue was that it expects the value to be in the options array at all times, so without this prop we'll need to add it back to the array and have it show even when searching for a different string. it's been a bit so i'll look into it again today

Reply:
I remember that; we had to do some fancy stuff with memorization to keep the references intact IIRC

Reply:
![Screen Shot 2022-04-25 at 9 35 11 AM](https://user-images.githubusercontent.com/11381794/165133739-c8212292-58ee-4067-a14f-261df466bcd8.png)

here's the bug @mastercactapus- I'll work towards a fix this afternoon

Reply:
should be resolved now 👍